src/main/java/uk/gov/hmcts/reform/civil/enums/CaseCategory.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.civil.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CaseCategory {
+    UNSPEC_CLAIM,
+    SPEC_CLAIM;
```

Comment:
Im not sure where we use this - but similar to an unspec claim, we need to set this for Specific claims.
They currently set the super claim type, which I believe is to be migrated / made redundant when we release Access Profiles service code..

Part of this PR should be to convert any usages of superClaimType - to instead use the Case Access Category field in case data